parser.py

```diff
@@ -121,7 +130,7 @@ def filter_only_unique(
             for elem in data.get("warnings")
```

Comment:
I've reverted this back to the original code - that's the only way it works for me, with the mock inputs and outputs from c58f390cc802db551bd3963b27f4a774071f4c00

Reply:
Perfect! Now it works for me.

<img width="1133" alt="image" src="https://github.com/user-attachments/assets/7c524171-0ad9-4461-af1e-ae570d6539ba" />